src/command_modules/azure-cli-network/azure/cli/command_modules/network/commands.py

```diff
@@ -31,7 +34,10 @@
 cli_command(__name__, 'network application-gateway list', custom_path.format('list_application_gateways'))
 cli_command(__name__, 'network application-gateway start', 'azure.mgmt.network.operations.application_gateways_operations#ApplicationGatewaysOperations.start', cf_application_gateways)
 cli_command(__name__, 'network application-gateway stop', 'azure.mgmt.network.operations.application_gateways_operations#ApplicationGatewaysOperations.stop', cf_application_gateways)
-cli_command(__name__, 'network application-gateway show-backend-health', 'azure.mgmt.network.operations.application_gateways_operations#ApplicationGatewaysOperations.backend_health', cf_application_gateways)
+
+if get_api_version(ResourceType.MGMT_NETWORK) in ['2016-09-01']:
```

Comment:
This mechanism seems very burdensome. I doubt this command is going to vanish in APIs moving forward, and I shouldn't have to add to this (and every other) list for every new API version.  This mechanism needs to be improved with some kinds of convenience methods like "api_version_greater_than" or something.